src/modules/esl-tab/core/esl-tabs.ts

```diff
@@ -181,4 +196,44 @@ export class ESLTabs extends ESLBaseElement {
   protected _onResize = rafDecorator(() => {
     this._deferredFitToViewport(this.$current, 'auto');
   });
+
+  /** ESLMediaRuleList instance of the scrollable type mapping */
+  public get scrollableTypeRules() {
+    if (!this._scrollableTypeRules) {
+      this.scrollableTypeRules = ESLMediaRuleList.parse<string>(this.scrollable, ESLMediaRuleList.STRING_PARSER);
+    }
+    return this._scrollableTypeRules;
+  }
+  public set scrollableTypeRules(rules: ESLMediaRuleList<string>) {
+    if (this._scrollableTypeRules) {
+      this._scrollableTypeRules.removeListener(this._onScrollableTypeChange);
+    }
+    this._scrollableTypeRules = rules;
+    this._scrollableTypeRules.addListener(this._onScrollableTypeChange);
+  }
+
+  /** @returns current scrollable type */
+  public get currentScrollableType(): string {
+    return this.scrollableTypeRules.activeValue || '';
+  }
+
+  /** Handles scrollable type change */
+  @bind
+  protected _onScrollableTypeChange() {
+    this.updateScrollableType();
+  }
+
+  /** Update element state according to scrollable type */
+  protected updateScrollableType() {
+    ESLTabs.supportedScrollableTypes.forEach((type) => {
+      CSSUtil.toggleClsTo(this, `${type}-alignment`, this.currentScrollableType === type);
+    });
+    this.fitToViewport(this.$current);
+
+    if (this.currentScrollableType === 'disabled') {
+      this.unbindScrollableEvents();
+    } else {
+      this.bindScrollableEvents();
```

Comment:
Please remove the same binding from the connected callback, its better to have one flow to initiate this,